tests/python/test_mpm88.py

```diff
@@ -61,7 +61,7 @@ def substep():
             w = [
                 0.5 * (1.5 - fx)**2, 0.75 - (fx - 1.0)**2, 0.5 * (fx - 0.5)**2
             ]
-            new_v = ti.Vector.zero(ti.f32, 2)
+            new_v = ti.Matrix.zero(ti.f32, 2)
```

Comment:
```suggestion
            new_v = ti.Vector.zero(ti.f32, 2)
```

Reply:
```py
>           new_v = ti.Vector.zero(ti.f32, 2)
E           AttributeError: 'function' object has no attribute 'zero'
```
Thanks to the fact that Vector is not a subclass of Matrix, but a initializer wrapper function.

Reply:
Now, I use `Vector.zero = Matrix.zero` solved the problem for now before we finally come to `ti.linalg.zero`.

Reply:
Sounds good. We should not use the subclass approach. Let's use add different initializers later.